src/gui/plugins/component_inspector/ComponentInspector.qml

```diff
@@ -227,6 +247,99 @@ Rectangle {
     }
   }
 
+  Dialog {
+    id: addSystemDialog
+    modal: false
+    focus: true
+    title: "Add System"
+    closePolicy: Popup.CloseOnEscape
+
+    ColumnLayout {
+      GridLayout {
+        columns: 2
+        columnSpacing: 30
+        Text {
+          text: "Name"
+          Layout.row: 0
+          Layout.column: 0
+        }
+
+        TextField {
+          id: nameField
+          selectByMouse: true
+          Layout.row: 0
+          Layout.column: 1
+          Layout.minimumWidth: 250
+          onTextEdited: {
+            addSystemDialog.updateButtonState();
+          }
+        }
+
+        Text {
+          text: "Filename"
+          Layout.row: 1
+          Layout.column: 0
+        }
+
+        TextField {
+          id: filenameField
+          selectByMouse: true
+          Layout.row: 1
+          Layout.column: 1
+          Layout.minimumWidth: 250
+          onTextEdited: {
+            addSystemDialog.updateButtonState();
+          }
+        }
+      }
+
+      Text {
+        id: innerxmlLabel
+        text: "Innerxml"
```

Comment:
Looks better on the UI

```suggestion
        text: "Inner XML"
```

Reply:
[da4f16a](https://github.com/gazebosim/gz-sim/pull/1549/commits/da4f16a0067dcce3bd0ae96d543786058646effc)